### ISSUE TITLE
kernels missing

### ISSUE THREAD

**Issue by cmdoret** [koszullab/instaGRAAL#1]
When installing instagraal v 0.1.2 on Ubuntu 18 through pip, the folder kernels is missing from the package. This results in the following error:

```
Traceback (most recent call last):
  File "/home/cmatthey/.local/bin/instagraal", line 11, in <module>
    sys.exit(main())
  File "/home/cmatthey/.local/lib/python3.6/site-packages/instagraal/instagraal.py", line 2108, in main
    output_folder=output_folder,
  File "/home/cmatthey/.local/lib/python3.6/site-packages/instagraal/instagraal.py", line 174, in __init__
    output_folder=output_folder,
  File "/home/cmatthey/.local/lib/python3.6/site-packages/instagraal/simu_single.py", line 174, in __init__
    self.gl_size_im,
  File "/home/cmatthey/.local/lib/python3.6/site-packages/instagraal/cuda_lib_gl_single.py", line 297, in __init__
    self.loadProgram(kernel_adapt_entry_point)
  File "/home/cmatthey/.local/lib/python3.6/site-packages/instagraal/cuda_lib_gl_single.py", line 1957, in loadProgram
    f = open(filename, "r")
FileNotFoundError: [Errno 2] No such file or directory: '/home/cmatthey/.local/lib/python3.6/site-packages/instagraal/kernels/kernel_sparse_adapt.cu'
```

Output of 
`ls -1 $HOME/.local/lib/python3.6/site-packages/instagraal/`
cuda_lib_gl_single.py
fragment.py
glutil.py
gpustruct.py
init_nuisance.py
__init__.py
instagraal.py
leastsqbound.py
log.py
optim_rippe_curve_update.py
parse_info_frags.py
__pycache__
pyramid_sparse.py
simu_single.py
vector.py
version.py

**Comment by baudrly** [comment]
Fixed by 85708c4827b6447127230b13aa3899e33dfa57a0

**Comment by michieitel** [comment]
Hi!

I get the same issue:

> ubuntu@lorean:~/tools/instaGRAAL/test_data$ instagraal hic_folder test_assembly.fasta test_output
> INFO :: filtering already done...
> WARNING :: /usr/local/lib/python3.6/dist-packages/instagraal-0.1.6-py3.6.egg/instagraal/pyramid_sparse.py:122: H5pyDeprecationWarning: The default file mode will change to 'r' (read-only) in h5py 3.0. To suppress this warning, pass the mode you need to h5py.File(), or set the global default h5.get_config().default_file_mode, or set the environment variable H5PY_DEFAULT_READONLY=1. Available modes are: 'r', 'r+', 'w', 'w-'/'x', 'a'. See the docs for details.
>   pyramid_handle = h5py.File(hdf5_pyramid_file)
> 
> INFO :: level already built...
> INFO :: level already built
> INFO :: level already built
> INFO :: level already built
> INFO :: level already built
> INFO :: level already built
> INFO :: level already built
> INFO :: level already built
> INFO :: level already built
> INFO :: pyramid built.
> INFO :: init pyramid
> WARNING :: /usr/local/lib/python3.6/dist-packages/instagraal-0.1.6-py3.6.egg/instagraal/pyramid_sparse.py:1532: H5pyDeprecationWarning: The default file mode will change to 'r' (read-only) in h5py 3.0. To suppress this warning, pass the mode you need to h5py.File(), or set the global default h5.get_config().default_file_mode, or set the environment variable H5PY_DEFAULT_READONLY=1. Available modes are: 'r', 'r+', 'w', 'w-'/'x', 'a'. See the docs for details.
>   self.data = h5py.File(self.pyramid_file)
> 
> INFO :: pyramid loaded
> INFO :: loading data from level = 4
> INFO :: import reference genome
> INFO :: loading data from level = 3
> INFO :: mean frag area = 704.387939453125
> INFO :: N frag duplicated = 0
> INFO :: MAX ID CONTIG = 422
> INFO :: total mem used by sparse data = 199.942432
> INFO :: loading kernels ...
> Traceback (most recent call last):
>   File "/usr/local/bin/instagraal", line 11, in <module>
>     load_entry_point('instagraal==0.1.6', 'console_scripts', 'instagraal')()
>   File "/usr/local/lib/python3.6/dist-packages/instagraal-0.1.6-py3.6.egg/instagraal/instagraal.py", line 1214, in main
>     output_folder=output_folder,
>   File "/usr/local/lib/python3.6/dist-packages/instagraal-0.1.6-py3.6.egg/instagraal/instagraal.py", line 159, in __init__
>     output_folder=output_folder,
>   File "/usr/local/lib/python3.6/dist-packages/instagraal-0.1.6-py3.6.egg/instagraal/simu_single.py", line 168, in __init__
>     self.pos,
>   File "/usr/local/lib/python3.6/dist-packages/instagraal-0.1.6-py3.6.egg/instagraal/cuda_lib_gl_single.py", line 294, in __init__
>     self.loadProgram(kernel_adapt_entry_point)
>   File "/usr/local/lib/python3.6/dist-packages/instagraal-0.1.6-py3.6.egg/instagraal/cuda_lib_gl_single.py", line 1952, in loadProgram
>     f = open(filename, "r")
> FileNotFoundError: [Errno 2] No such file or directory: '/usr/local/lib/python3.6/dist-packages/instagraal-0.1.6-py3.6.egg/instagraal/kernels/kernel_sparse_adapt.cu'
> 

I installed all like this:

> # libraries:
> sudo apt-get install libboost-all-dev
> sudo apt-get install hdf5-tools
> sudo apt-get install libglu1-mesa
> sudo apt-get install libxi-dev
> sudo apt-get install libxmu-dev
> sudo apt-get install libglu1-mesa-dev
> sudo apt-get install freeglut3-dev
> 
> # python dependencies:
> sudo pip3 install numpy
> sudo pip3 install scipy
> sudo pip3 install matplotlib
> sudo pip3 install codepy
> sudo pip3 install h5py
> sudo pip3 install pyopengl
> sudo pip3 install docopt
> sudo pip3 install biopython
> 
> sudo pip3 install --upgrade --no-cache-dir -e git+https://github.com/inducer/codepy.git@master#egg=codepy
> 
> 
> 
> 
> git clone --recurse-submodules https://github.com/inducer/pycuda.git
> cd pycuda
> python3.6 configure.py --cuda-enable-gl --no-use-shipped-boost
> sudo python3.6 setup.py install
> 
> ## instaGRAAL:
> cd ..
> git clone https://github.com/nowanda92/instaGRAAL
> cd instaGRAAL
> sudo python3.6 setup.py install


output of ls -1 /usr/local/lib/python3.6/dist-packages/instagraal-0.1.6-py3.6.egg/instagraal/:

> cuda_lib_gl_single.py
fragment.py
glutil.py
gpustruct.py
init_nuisance.py
__init__.py
instagraal.py
leastsqbound.py
linkage.py
log.py
optim_rippe_curve_update.py
parse_info_frags.py
__pycache__
pyramid_sparse.py
simu_single.py
vector.py
version.py
